### ISSUE TITLE
Python3 error

### ISSUE THREAD

**Issue by cyphunk** [mrts/burp-suite-http-proxy-history-converter#5]
Having issue parsing example file:

```
$ python convert-burp-suite-http-proxy-history-to-csv.py  example/burp-http-history.xml 
Traceback (most recent call last):
  File "/usr/lib/python3.6/site-packages/xmltodict.py", line 309, in parse
    parser.ParseFile(xml_input)
TypeError: read() did not return a bytes object (type=str)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "convert-burp-suite-http-proxy-history-to-csv.py", line 207, in <module>
    main()
  File "convert-burp-suite-http-proxy-history-to-csv.py", line 23, in main
    http_history = parse_http_history(args.filename)
  File "convert-burp-suite-http-proxy-history-to-csv.py", line 83, in parse_http_history
    return xmltodict.parse(f)
  File "/usr/lib/python3.6/site-packages/xmltodict.py", line 311, in parse
    parser.Parse(xml_input, True)
TypeError: a bytes-like object is required, not '_io.TextIOWrapper'
```

**Comment by mrts** [comment]
Thanks for the report! Does your pull request #6  fix this?

**Comment by cyphunk** [comment]
yes pull#6 fixes this issue for me for me (python 3.6)
